lib/Menu.rb

```diff
@@ -0,0 +1,38 @@
+require_relative 'Menu'
+
+class Menu
+
+	DEFAULT_MENU = { "Margarita" => 6.00, 
+	        "Chorizo" => 7.00, 
+	        "Napolitana" => 7.00, 
+	        "Pepperoni" => 7.00,
+	        "Wild Mushroom" => 7.00,
+	        "Sausage & Wild Broccolli" => 7.00 }
+
+	def see_menu
+		DEFAULT_MENU
+	end
+
+	def order(item, amount)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `item`. If it's necessary, use `_` or `_item` as an argument name to indicate that it won't be used. You can also write as `order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)<br>Lint/UnusedMethodArgument: Unused method argument - `amount`. If it's necessary, use `_` or `_amount` as an argument name to indicate that it won't be used. You can also write as `order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)